apps/icons/src/ChevronLeft12.tsx

```diff
@@ -15,7 +15,7 @@ const SvgChevronLeft12 = (props: SVGProps<SVGSVGElement>) => (
       strokeLinecap="round"
       strokeLinejoin="round"
       strokeWidth={1.5}
-      d="M8 3 5 6l3 3"
+      d="M7 3L4 6L7 9"
```

Comment:
Did you change the source file too and regenerate?

Reply:
I am taking this from FIGMA. Do we need to change else where for this? The currently Chevron is not the correct asset (it is a bit slanted)

Reply:
These are generated from the SVG assets, look at the readme of this directory, it should tell you how to get going. But in general, you should mass export from the icons figma file, then run clean + generate.